### PR TITLE
[Eventonica H1-2021] Update all relevant files to be consistent with current schedule

### LINKED ISSUE #1475: [Eventonica] Replace Event Recommender with Eventonica
In Eventonica's starter code (in [the folder](https://github.com/Techtonica/curriculum/tree/main/projects/eventonica/starter-code) +[Part 1](https://github.com/Techtonica/curriculum/blob/main/projects/eventonica/eventonica-part1-objects.md#starter-code)) the three main classes are  `Eventonica`, `Event`, and `User`.

In Part 1's starter code description and in all other parts, the text refers to the three main classes as `EventRecommender`, `Event`, and `User`. 

Given the changed description in 2f4affaaedc4819754c9a6cbba22f7129803386e and original files' starter code cf1ba07d97688d25dfbb645129b6857d47db0e06, should all mentions of `EventRecommender` be updated to `Eventonica`?

### LINKED ISSUE #1480: Invalid file link 404 error line 19
location of file: 
curriculum/projects/eventonica/eventonica-part7-testing.md

### PR DESCRIPTION
**Pull request by vegetabill**
- Restore any renamed files with their old name and with "redirect" instructions to find the updated content, if applicable
  - Restores the starter jasmine spec and aligns it with other starter code
  - Removes use of static since that was not part of the curriculum
  - resolves #1475 due to "main" class change from EventRecommender => Eventonica
  - resolves #1480 to fix link from testing to starter code

phew

I QA'd this against the Monday.com schedule and I _think_ that everything is okay for the rest of the cohort, Eventonica-wise.

### CODE REVIEW

File: projects/eventonica/eventonica-part6-postgres.md [hunk 1]
```diff
@@ -94,9 +95,11 @@ TL;DR - they are taking their in-memory backend data objects from Part 5 and usi
 
 - README should contain instructions on how to load the testing database schema (likely with data)
 - A big part of reviewing this is checking it out and making sure it works
-- I've been using a new [beta GitHub CLI](https://cli.github.com/) that can quickly checkout a PR
 - README should also mention how to run any tests
-- SQL commands should be in the EventRecommender "DAO" object, not in the Express app route handlers
-- If the code is all stuffed into the handlers, send your preferred explanatory link about the concept of system layers
+- SQL commands should be in the model objects, not in the Express app route handlers
+  - If the code is all stuffed into the handlers, send your preferred explanatory link about the concept of system layers. Our curriculum doesn't currently have a lesson for it.
 - If there are no unit tests or API tests, flag that as an area of possible improvement
```
Comment (naomiquinones) [inline projects/eventonica/eventonica-part6-postgres.md:101]: @SarahCawley, any thoughts on how to handle this?

**Review by naomiquinones — DISMISSED** [review]
@vegetabill, thanks for all your work on this! I noted a few minor typos and possible name change.

**Comment by vegetabill** [comment]
Don't want to start another week with the mismatch, so going to merge this but feel free to review post-merge and I will follow up with additional fixes.

Thanks @naomiquinones

File: projects/eventonica/eventonica-part6-postgres.md (hunk 1 above)
Comment (vegetabill) [reply projects/eventonica/eventonica-part6-postgres.md:101]: IIRC this was instructions to the volunteer assignment graders actually. May or may not be a feature request.

Comment (naomiquinones) [reply projects/eventonica/eventonica-part6-postgres.md:101]: @vegetabill, thanks for clarifying!

**Pull request merged**